tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -515,6 +515,7 @@
   "DD_IAST_REGEXP_TIMEOUT": "iast_regexp_timeout",
   "DD_IAST_TELEMETRY_VERBOSITY": "iast_telemetry_verbosity",
   "DD_IAST_TRUNCATION_MAX_VALUE_LENGTH": "iast_truncation_max_value_length",
+  "DD_IAST_DB_ROWS_TO_TAINT": "iast_db_rows_to_taint",
```

Comment:
Just wondering if this was added to `dd-go` as well?

Reply:
Yes it was. The PR is in the description 👍